server/container/guice/cassandra-guice/src/test/java/org/apache/james/JamesCapabilitiesServerTest.java

```diff
@@ -86,6 +85,21 @@ public void startShouldFailWhenNoMoveCapability() throws Exception {
     }
     
     @Test
+    public void startShouldFailWhenNoRightSearchCapability() throws Exception {
+        MailboxManager mailboxManager = mock(MailboxManager.class);
+        when(mailboxManager.getSupportedMailboxCapabilities())
+            .thenReturn(EnumSet.complementOf(EnumSet.of(MailboxManager.MailboxCapabilities.ACL)));
+        when(mailboxManager.getSupportedMessageCapabilities())
+            .thenReturn(EnumSet.allOf(MailboxManager.MessageCapabilities.class));
+        when(mailboxManager.getSupportedSearchCapabilities())
+            .thenReturn(EnumSet.allOf(MailboxManager.SearchCapabilities.class));
+
+        server = createCassandraJamesServer(mailboxManager);
```

Comment:
This test is not clear for me you say 'WhenNoRightSearchCapability" but on your when you return the "ACL" capacities